about-blaze.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+<style>
+  body{
+        background-image:url("https://cdn.glitch.com/7e934bf3-11ca-4ed8-bb0e-af5a28599d9f%2Fmobile%20green%20background.png?v=1604358440560");
+        background-size:100%;
+      }
+    <title>About Blaze Pizza</title>
+    </style>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="about-blaze.css">
+  </head>
+  <body>
+ <div>
+   <img class="logo"
+        src="https://cdn.glitch.com/7e934bf3-11ca-4ed8-bb0e-af5a28599d9f%2F24403.png?v=1604357916769"></div>
+   <div id="mySidepanel" class="sidepanel">
+      <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">×</a>
+      <a href="1.html">Home</a>
+      <a href="new-pizza.html">New Pizzas</a>
+      <a href="about-blaze.html">Blaze Pizza</a>
+      <a href="about-us.html">About us</a>
+    </div>
+    
+    <button class="openbtn" onclick="openNav()">☰</button> 
+    <h3 class="title">
+      Blaze Pizza?
+    </h3>
+  <img class="stove"
+      src="https://cdn.glitch.com/7e934bf3-11ca-4ed8-bb0e-af5a28599d9f%2F3.png?v=1604435164968">
+<p class="first-paragraph">
+      Intelligent Choices for our Pizzas, People and Planet.
+We believe that each of us can benefit from making smarter decisions about what’s in our food, how we treat each other, and how we protect this wonderful planet we all share. How’s that work? Glad you asked.
+    </p>
+    <p class="second-paragraph">
+      Real Ingredients.
+We believe in Here and Now. To us, that means fresh dough, NEVER FROZEN, made in-house each day simply from unbleached flower, filtered water, extra-virgin olive oil, salt, and a touch of sugar. No chemicals. No additives. No kidding. We’ve also nixed the nitrites from our cured meats, switched to “true hue” banana peppers and olives without artificial colors, and created our own salad dressings, leaving the artificial preservatives behind.
+    </p>
+    <img class="pizza-in-pan"
+      src="https://cdn.glitch.com/7e934bf3-11ca-4ed8-bb0e-af5a28599d9f%2F1.png?v=1604435097541">
+      <img class="pizza"
+      src="https://cdn.glitch.com/7e934bf3-11ca-4ed8-bb0e-af5a28599d9f%2F2.png?v=1604435160232">
+    <p class="third-paragraph">
+      ECO-Friendly Packaging.
+We believe that the decisions we make for the Earth are extremely important, too. That’s why we’ve gone to great lengths to make sure that, wherever practical, our packaging is recyclable, compostable, and/or made from post-consumer reclaimed materials.
+    </p>
+   <script> function openNav() {
+      document.getElementById("mySidepanel").style.width = "250px";
+    }
+    
+    function closeNav() {
+      document.getElementById("mySidepanel").style.width = "0";
+    }
+
+</script>
```

Comment:
It looks like you're repeating this a lot. If you put it in one scripts.js file that's available on all pages, you only need to write it once - and only have one place to fix it if something ends up being buggy!